code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -1393,10 +1394,10 @@ obj/item/gun/ballistic/automatic/bar
 	burst_shot_delay = 1.5
 	is_automatic = TRUE
 	automatic = 1
-	autofire_shot_delay = 2.5
+	burst_shot_delay = 1
 	fire_delay = 4
 	extra_damage = 25
-	spread = 8
+	spread = 20 //twice as much as the minigun, because it amuses me
```

Comment:
Why would it be higher than minigun?

Reply:
Because it amuses him. His words.

Reply:
> Because it amuses him. His words.

Not a good answer.

Reply:
Because it amuses me. That's my answer.

Also, because it fires at the same rate as the minigun but does significantly more damage, and the legion is meant to be close range anywya.

Also, because the ORIGINAL centurion M1919 on Bad Deathclaw had roughly this amount of spread for its ridiculous burst, and it was very funny.